readme.md

```diff
@@ -207,24 +223,56 @@ class Person
 end
 
 class LoudPerson < Person
-  def introduce
+  def introduction
     puts "HELLO, I'M #{name.upcase}"
   end
 end
 
-adam = Person.new("Adam", 10)
+jill = Person.new("Jill", 10)
 bob = LoudPerson.new("Bob", 10)
 
 puts bob.name
-puts bob.introduce
+puts bob.introduction
 
 bob.hunger_level = 5
 puts bob.hunger_level
 ```
 
+### You Do: Inheritance
+
+- Make a sleepy person, who sleeps through the introduction.
+- Make a baby, who can only say "Dada" and is always hungry (hunger_level never reaches 0)
+
+
+
+## Visibility
+
+When discussing object-oriented programming, you may hear the term "visibility".  This refers to the availability, or scope, of a method. Who can call this method on the object?
+
+- Public: any one can call this method.  Public is default accessibility level for class methods.
+- Private and Protected: (not common) Not visible to other objects from other classes.  Control visibility to other instances of that Class, including descendants.
+
 
 ## Sample Questions
 
 - Explain the use of self in Ruby
 - Explain the difference between local, instance and class variables
 - Define and differentiate between class and instance methods
+-
+
+## Vocabulary
+
+* **class**: an object blueprint
+* **instance**: a constructed class instance
+* **sub-class**: a class derived through inheritance.
+* **extends**: a class is "extended" into a sub-class.
```

Comment:
I really like the terms cheat sheet here at the end! I think its really useful when going back to review previous lesson plans as a student